build/nse/vl3-nse/Dockerfile.no_go_build_ucnf

```diff
@@ -14,3 +14,4 @@ COPY vl3_nse /bin/cnf-vppagent
 
 COPY etc/ /etc/
 COPY etc/supervisord/supervisord.conf /opt/vpp-agent/dev/supervisor.conf
+COPY etc/supervisord/init_hook.sh /usr/bin/init_hook.sh
```

Comment:
we'd need the same sort of change for the kiknos NSE, right?

Reply:
Right, but with a bit different content. Will add that into this PR

Reply:
Done, also added a proper `supervisord.conf` for kiknos into the Kiknos NSE Docker image. It is normally overridden by the helm chart, but I think it is a good idea to have it backed in the image for potential usage without Kiknos helm chart.